tests/php/Control/SessionTest.php

```diff
@@ -16,7 +16,11 @@
 use SilverStripe\Core\Config\Config;
 use SilverStripe\Core\Injector\Injector;
 use PHPUnit\Framework\Attributes\DataProvider;
+use PHPUnit\Framework\Attributes\PreserveGlobalState;
```

Comment:
Noticed this `use` statement was missing... I have no idea why it wasn't causing CI failures but this class is referenced in some of the tests higher up in this class.